metrics/Code_Changes.md

```diff
@@ -101,7 +103,7 @@ Some useful visualizations are:
 These could be represented as bar charts, with time running in the X axis.
 Each bar would represent a code changes during a certain period (eg, a month).
 
-## Reference Implementation
+## Sample Implementation
```

Comment:
I would stick to "Reference", in the sense that if you're implementing, this is not just another implementation, that you can use as a sample, but a the reference implementation, that you should have into account, in the sense that the results of your implementation should be the same than this one. This is the whole idea of having reference implementations.

Reply:
Thanks @jgbarah. The current template uses Sample. Now, I'm not saying that that's right :) I certainly see your point and moving Sample --> Reference in the other WGs wouldn't be difficult.